testgridshot

```diff
@@ -251,24 +278,35 @@ shoot() {
 
         local_log "done, image will be available at ${image_url}"
       )&
+      pids+=( $! )
     done
   done
 
-  wait
+  wait_for_all "${pids[@]}"
 
   if [ "$idx" -lt 1 ]; then
-    log "No screenshots captured for ${target_release} ... ${SHRUG}"
-    return
+    {
+      echo
+      echo "**NO $(comma_sep "${STATES}") TESTS FOUND**"
```

Comment:
Can we tweak the language here? 
"No `foo`, `bar` tests found" could be interpreted as "We couldn't find the tests".

In contrast with "No failing tests found for `foo`, `bar` dashboards", which leaves little to interpretation.

ref: https://github.com/kubernetes/release/issues/965#issuecomment-564741165

Reply:
It ends up being, depending on the setting [`STATES`](https://github.com/kubernetes/release/blob/f033a8215f94913277976c4a3cc6f130a4e24cb7/testgridshot#L34-L36)
> NO FAILING, FLAKY TESTS FOUND
or
> NO PASSING TESTS FOUND

I agree, that the word "FOUND" might indicate "We couldn't find the tests".